linker/shared/src/main/scala/org/scalajs/linker/backend/wasmemitter/ClassEmitter.scala

```diff
@@ -441,6 +442,105 @@ class ClassEmitter(coreSpec: CoreSpec) {
     }
   }
 
+  /** Generates the array classes. */
```

Comment:
```suggestion
  /** Generates the array class types and typeData for primitives and jl.Object. */          
```

Maybe something like this explains it more clearly?